src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Profile.cshtml

```diff
@@ -0,0 +1,116 @@
+﻿@page "{UserName?}"
+@model APIViewWeb.Pages.Assemblies.ProfileModel
+@using APIViewWeb.Helpers
+@using APIViewWeb.Models
+@{
+    if(Model.User.GetGitHubLogin().Equals(Model.userProfile.UserName)) 
+    {
+        ViewData["Title"] = "Your profile";
+    }
+    else 
+    {
+        ViewData["Title"] = Model.userProfile.UserName;
+    }
+    TempData["UserPreference"] = PageModelHelpers.GetUserPreference(Model._preferenceCache, User.GetGitHubLogin()) ?? new UserPreferenceModel();
+
+    List<string> themes = new List<string>()
+    {
+        "light-theme",
+        "dark-theme",
+        "dark-solarized-theme"
+    };
+    string[] Languages = { "c", "c#", "c++", "go", "java", "javascript", "python", "protocol", "swift" };
```

Comment:
I would prefer to avoid maintaining a list of languages in one more place. List is already missing few options that we show in languages dropdown on main page.

Reply:
One another change. These options are not in proper casing. It should be C, C#, C++, CADL, Go, Java, JavaScript, Python, Swagger, Swift

Reply:
Is there somewhere this list is currently maintained that we can use to populate this dropdown? (eg, the configuration?)

Reply:
Language dropdown pulls directly from the database. https://github.com/Azure/azure-sdk-tools/blob/b009cdc5e2dcaf73d3d9020f4aca1a2a269ad700/src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml.cs#L68

Reply:
@chidozieononiwu The language dropdown on Index.cshtml only gets languages that currently have a review, instead of all reviewable languages - is there an alternative that allows us to get all reviewable languages?

Reply:
That does not exist at the moment. Going the configuration route is not a bad idea, but that may still need to be updated at some point so am not sure how it's different form getting existing reviews.

Reply:
@praveenkuttappan @chidozieononiwu Can you please give your recommendation on how best to have a full set of supported languages? If we start a fresh APIView, we can't configure reviewers for a language until someone has uploaded a review for that language, which seems less than ideal :-)

Reply:
For now we are hard-coding the pretty list, and Praveen will improve this at a future time.

Reply:
Do we have (create) an issue for this follow up?